CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ### 🔄 Changed
+- An error message about recovery request mode was incorrectly logged after web-socket connect [#3426](https://github.com/GetStream/stream-chat-swift/pull/3426)
```

Comment:
This sounds like it should belong to the Fixed section 🤔 Also, we are missing the ## StreamChat section

Reply:
Fixed